CONTRIBUTING.md

```diff
@@ -2,58 +2,57 @@
 
 This page provides information about contributing code to the Jenkins core codebase.
 
-:exclamation: There's a lot more to the Jenkins project than just code. For information on contributing to the Jenkins project overall, check out [Participate].
+:exclamation: There's a lot more to the Jenkins project than just code. For information on contributing to the Jenkins project overall, here's a page on how to [Participate].
 
 ## Getting started
 
 1. Fork the repository on GitHub
 2. Clone the forked repository to your machine
-3. Install the development tools. In order to develop Jenkins, you need the following tools:
+3. Install the necessary development tools. In order to develop Jenkins, you need the following:
   * Java Development Kit (JDK) 8 or 11.
-    In Jenkins project we usually use [OpenJDK](http://openjdk.java.net/) or [AdoptOpenJDK](https://adoptopenjdk.net/), but you can use other JDKs as well.
+    In the Jenkins project we usually use [OpenJDK](http://openjdk.java.net/) or [AdoptOpenJDK](https://adoptopenjdk.net/), but you can use other JDKs as well.
     * For JDK 11 there might be some compatibility issues in developer tools,
       please see [this page](https://wiki.jenkins.io/display/JENKINS/Java+11+Developer+Guidelines#Java11DeveloperGuidelines-Knowndevelopertoolsissues) for more info.
       If you hit a new issue, please report it with a `java11-devtools-compatibility` label in our issue tracker.
```

Comment:
Suggestion -> If you find a new issue...

Reply:
```suggestion
      If you find a new issue, please report it with a `java11-devtools-compatibility` label in our issue tracker.
```